CHANGELOG.md

```diff
@@ -6,10 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [1.2.0-rc.2] - 2024-06-11
+
+> TRUE or FALSE, it's a problem!
 
+### Important Notes
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
    import libcontainer/cgroups/devices. (#3452, #4248)
+ * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
+   (see #4233 for more details)
+
+### Added
+
+ * CI: add actuated-arm64. (#4142)
+
+### Fixed
+
+ * Fix tests in debian testing (mount_sshfs.bats) - minimal fix. (#4245)
+ * Set the default value of CpuBurst to nil instead of 0. (#4211)
+ * Fix set nofile rlimit error. (#4265, #4282, #4290)
+ * fix checkpoint/restore tests on actuated-arm64. (#4276)
+ * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4258)
+ * Fix codespell warnings. (#4291)
+ * libct/cg/dev: fix TestSetV1Allow panic. (#4295)
+ * tests/int/scheduler: require smp. (#4298)
+ * tests/int: swap-related fixes. (#4188)
+ * script/*: fix gpg usage wrt keyboxd. (#4189)
+ * fix a debug msg for user ns in nsexec. (#4311)
+ * libct/cg: write unified resources line by line. (#4186)
+
+### Changed
+
+ * ci/test: exclude some runc_nodmz jobs. (#4238)
+ * libcontainer: force apps to think fips is enabled/disabled for testing. (#4246)
+ * Set temporary single CPU affinity before cgroup cpuset transition.. (#3923)
```

Comment:
This one is reverted in https://github.com/opencontainers/runc/pull/4283 and thus should not be listed.